Merge Theros Challenge Deck creatures into token.xml (#225)
Cockatrice/Magic-Token: README.md

This repo contains token information in [Cockatrice](https://github.com/cockatrice/cockatrice)'s [XML card database format](https://github.com/Cockatrice/Cockatrice/wiki/Custom-Cards-&-Sets#to-add-your-own-custom-cards-follow-these-steps) for Magic: The Gathering.<br />
It describes [![](https://img.shields.io/badge/dynamic/xml.svg?label=&colorB=white&url=https%3A%2F%2Fraw.githubusercontent.com%2FCockatrice%2FMagic-Token%2Fmaster%2Ftokens.xml&query=count(%2F%2Fcard))](https://github.com/Cockatrice/Magic-Token/blob/master/tokens.xml) individual tokens with linked art and the relation to their creators.

> A copy of the tokens file included with Cockatrice can be found here: [**tokens.xml**](https://raw.githubusercontent.com/Cockatrice/Magic-Token/master/tokens.xml)

Reviewer: ```suggestion
Cockatrice pulls its copy of the token file from here: [**tokens.xml**](https://raw.githubusercontent.com/Cockatrice/Magic-Token/master/tokens.xml)
```

A bit shorter version to make it fit a single line.

Author: Thank you for updating the CI check; I was a little worried that that failing check would scare reviewers off, but I wasn't sure how to (or if I even could) change it myself. 

And, yes, I only moved the creature spells as the rest are in cards.xml already. It seems odd to have them in two places (either in both cards.xml and tokens.xml, or in both cards.xml and challenge_tokens.xml) since that creates redundant entries in the client. However, I could understand not wanting to give up control of the non-creature spells and leave them in the hands of whatever processes create cards.xml. It would probably also be fine to put all challenge deck creature and non-creature spells in tokens.xml, but then the non-creature spells would have to be removed from cards.xml somehow, which I don't know how to do.

Author: Thank you for updating the CI check; I was a little worried that that failing check would scare reviewers off, but I wasn't sure how to (or if I even could) change it myself. 

And, yes, I only moved the creature spells as the rest are in cards.xml already. It seems odd to have them in two places (either in both cards.xml and tokens.xml, or in both cards.xml and challenge_tokens.xml) since that creates redundant entries in the client. However, I could understand not wanting to give up control of the non-creature spells and leave them in the hands of whatever processes create cards.xml. It would probably also be fine to put all challenge deck creature and non-creature spells in tokens.xml, but then the non-creature spells would have to be removed from cards.xml somehow, which I don't know how to do.
